support-config/src/main/scala/com/gu/support/config/AmazonPayConfig.scala

```diff
@@ -5,7 +5,7 @@ import com.typesafe.config.Config
 case class AmazonPayConfig(
   sellerId: String,
   clientId: String
-) extends TouchpointConfig
```

Comment:
Removing the requirement for these config classes to extend `TouchpointConfig` means that payment-api doesn't need to depend on support-config which is not really relevant for it.